my_game.toml

```diff
@@ -62,3 +62,5 @@ TITLE_X = 400  # pCha
 TITLE_Y = 450  # px
 BUTTON_X = 400  # px
 BUTTON_Y = 300  # px
+
+RESTART_BUTTON = 114  # R key
```

Comment:
Can we rename to _UI_RESTART_KEY_ to match naming of other variables for defining keys.